android_beta/8.2beta.yml

```diff
@@ -0,0 +1,84 @@
+release:
+  release_date: '2024-11-18'
+  text: |
+    **These notes apply to Thunderbird for Android version 8.2 betas, first released November 18th, 2024**
+
+  groups:
+    - ' '
+
+  releases:
+    - version: '8.2b1'
+      release_date: '2024-11-18'
+
+notes:
+  ## Beta 1
+  # New
+  - note: Change account initials to use display name for better customizability
+    tag: new
+    group: 1
+
+  - note: The account icons no longer switch positions when selected
+    tag: new
+    group: 1
+
+  - note: Improve messaging for folder notification setup and clarify any missing requirements
```

Comment:
I don't understand what this is about.

Reply:
This was https://github.com/thunderbird/thunderbird-android/pull/8525
I didn't want to get too technical, but admit my version is pretty vague

Reply:
This should probably go into the "fixed" section. Suggestion for wording:
> Push service sometimes wasn't started when it should have been

Reply:
How about

Indicate if push permissions requirements are missing in the system notification

Reply:
That functionality is not new. The issue that was fixed is that the push service wasn't started and thus the message wasn't displayed.

Reply:
Apologies, our messages overlapped. Thanks for providing the message, I'm happy to use that.